From eebcde949f7b1ea699edb591363c9985a55ba77c Mon Sep 17 00:00:00 2001
From: "Luke D. Jones" <luke@ljones.dev>
Date: Wed, 10 Aug 2022 09:45:15 +1200
Subject: [PATCH 02/17] Fixes: 382b91db8044 ("asus-wmi: Add egpu enable
 method")

The egpu_enable attribute was not documented, this adds the
required documentation.

Signed-off-by: Luke D. Jones <luke@ljones.dev>
---
 Documentation/ABI/testing/sysfs-platform-asus-wmi | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/Documentation/ABI/testing/sysfs-platform-asus-wmi b/Documentation/ABI/testing/sysfs-platform-asus-wmi
index 0f932fd60f4a..b984bdcd0c40 100644
--- a/Documentation/ABI/testing/sysfs-platform-asus-wmi
+++ b/Documentation/ABI/testing/sysfs-platform-asus-wmi
@@ -65,4 +65,14 @@ Contact:       "Luke Jones" <luke@ljones.dev>
 Description:
                Disable discrete GPU:
                        * 0 - Enable dGPU,
-                       * 1 - Disable dGPU,
\ No newline at end of file
+                       * 1 - Disable dGPU,
+
+What:          /sys/devices/platform/<platform>/egpu_enable
+Date:          Aug 2022
+KernelVersion: 5.17
+Contact:       "Luke Jones" <luke@ljones.dev>
+Description:
+               Enable the external GPU paired with ROG X-Flow laptops.
+               Toggling this setting will also trigger ACPI to disable the dGPU:
+                       * 0 - Disable,
+                       * 1 - Enable,
\ No newline at end of file
-- 
2.37.1

